src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -881,12 +865,10 @@ GetSizeQueryBySizeQueryType(SizeQueryType sizeQueryType)
 
 /*
  * GenerateAllShardStatisticsQueryForNode generates a query that returns:
- * - all shard_name, shard_size pairs for the given node (if useShardMinMaxQuery is false)
- * - all shard_id, shard_minvalue, shard_maxvalue, shard_size quartuples (if true)
+ * all shard_name, shard_size pairs for the given node
```

Comment:
Do we return just `shard_name, shard_size` pair or `shard_id, shard_name, shard_size` tuple? It seems like the latter